test/perf/ParseBenchmarks.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import sys
+
+# Add compatible ops here
+OpsWithPerformanceBenchmarks = [
+    'Gemm',
+    'Conv'
+]
+
+ExecutablePath = "/workdir/onnx-mlir/build/Debug/bin/"

```

Comment:
This should be determined based on the configuration coming from cmake, so that it works regardless of where the script is run.

Reply:
Check code in RunONNXModel.py line 93. Maybe that can work here too? Or can be used as a basis for a solution

Reply:
Let me know if you need help